qemu/tests/cfg/multi_host.cfg

```diff
@@ -108,6 +108,12 @@
                     #Count of migration during file transfer.
                     migrate_count = 3
                     regain_ip_cmd = ""
+                - with_stress:
+                    only Linux
+                    only after_login_vm
```

Comment:
need to update it, as after_login_vm will be moved to the same variant of other tests